src/axom/inlet/examples/example1.lua

```diff
@@ -0,0 +1,23 @@
+-- a simple test to accompany sera_lua_test
+--serac-thermal-solver

```

Comment:
Remove the top two comments and add something like "Basic example of a simulation Input Deck"